packages/components/aside/src/aside.js

```diff
@@ -0,0 +1,76 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import css from './aside.css';
+
+customElementDefineHelper(
+	'ts-aside',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		firstUpdated() {
+			this.shadowRoot.querySelector('slot[name="footer"]').addEventListener('slotchange', this.footerSlot.bind(this));
+		}
+
+		static get properties() {
+			return {
+				title: { type: String, attribute: 'data-title', reflect: true },
+				visible: { type: Boolean, attribute: 'data-visible', reflect: true },
+				hasFoot: { type: Boolean },
+				dir: { type: String }
+			};
+		}
+
+		constructor() {
+			super();
+			this.title = '';
+			this.visible = false;
+			this.hasFoot = false;
+			this.dir = this.bodyHasRTL ? 'rtl' : 'ltr';
+		}
+
+		close(e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "e". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)